lib/adapters/parsers/index.js

```diff
@@ -28,13 +27,6 @@ module.exports = {
             });
         }
 
-        if (parser) {
-            return new parser(options);
-        } else {
-            throw errors.runtimeError('RT-INVALID-OPTION-VALUE',{
-                option: 'format',
-                supported: _.keys(parserLookup).join(', ')
-            });
-        }
+        return new parser(options);
```

Comment:
It seems to be simpler to keep this file the way it was and just throw an error in the grok parser initialize() function if you can't require grok.